pycausal_explorer/meta/_single_learner.py

```diff
@@ -37,8 +25,57 @@ def predict(self, X, w):
         check_is_fitted(self)
         return self.learner.predict(np.column_stack([X, w]))
 
+
+class SingleLearnerRegressor(SingleLearnerBase):
+    """
+    Implementation of the single learner model.
+
+    Regressor version, should be used with continuous data.
+
+    Uses a single provided model to predict outcome when under treatment, and when not.
+    Uses that to estimate treatment effect.
+
+    Parameters
+    ----------
+    learner: estimator object
+        base learner to use when predicting outcome. Should implement fit and predict methods.
+    """
+
+    def __init__(self, learner):
+        super().__init__(learner)
+        self._estimator_type = "regressor"
+
     def predict_ite(self, X):
         check_is_fitted(self)
         return self.predict(X, np.ones(shape=X.shape[0])) - self.predict(
             X, np.zeros(shape=X.shape[0])
         )
+
+
+class SingleLearnerLogistic(SingleLearnerBase):
```

Comment:
Maybe change it to SingleLearnerClassifier, to be more genreal. What do you think?